### ISSUE TITLE
When :referring a symbol, autocomplete only shows up when I've finished typing it

### ISSUE THREAD

**Issue by danielcompton** [cursiveclojure/cursive#884]
When I am referring a symbol in, the autocomplete only shows when I've finished typing. Reproduction instructions:

0. Add [compojure "1.1.6"] to your project.clj (or any library really)
1. Require the compojure.core namespace
2. Type `:refer [POS`
3. Notice that there is no autocompletion for POST
4. Type `T`
5. Notice that the autocomplete for POST now shows up.

This is the least helpful time for autocomplete :)

**Comment by cursiveclojure** [comment]
Yeah, this is because the symbol isn't imported properly until you type the name of it. What's really required is position-sensitive auto-complete - see the recent mailing list thread about this.

**Comment by danielcompton** [comment]
I've been following that thread keenly :)

**Comment by PetrGlad** [comment]
In recent versions there is autocompletion menu but it seems to gather symbols from all of the project EXCEPT ones in the namespace that is currently required :)

**Comment by madvas** [comment]
What's been progress on this? Proper autocomplete for `:refer` is so much needed
